tuna/fin_class.py

```diff
@@ -361,18 +370,36 @@ def parse_applicability(self, json_in):
       self.logger.error("JSON file returned from Fin is empty")
       return False
 
+    #break down commits into smaller packets
+    pack_i = 0
+    pack = []
+    all_packs = []
+    for elem in json_in:
+      pack.append(elem)
+      pack_i += 1
+      if pack_i == 100:
+        all_packs.append(pack)
+        pack = []
+        pack_i = 0
+    if pack:
+      all_packs.append(pack)
```

Comment:
Perhaps this can be refactored into a general purpose util function

Reply:
yes, this happened more than once.